pkg/solver/store/store.go

```diff
@@ -20,6 +20,13 @@ type GetJobOffersQuery struct {
 	// we use the DealID property of the jobOfferContainer to tell if it's been matched
 	NotMatched bool `json:"not_matched"`
 
+	// Active job offers are umatched or in an in-progress deal.
+	// This includes the DealNegotiating, DealAgreed, or ResultsSubmitted states.
+	Active bool `json:"in_progress"`
+
+	// Cancelled job offers are in a JobOfferCancelled or JobTimedOut state
+	Cancelled bool `json:"cancelled"`
```

Comment:
Ah I see, ones for including results the other is for clearing out the stale offers.  Perhaps we can change the name of this to `inactive` or `terminated` so that it doesn't clash with `IncludeCancelled`?

Reply:
Wrote some more thoughts about this above, we may be able to simplify it to only use `Cancelled`.